ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/forkchoice/TerminalPowBlockMonitor.java

```diff
@@ -95,6 +79,15 @@ public synchronized void start() {
     }
     specConfigBellatrix = maybeSpecConfigBellatrix.get();
 
+    final boolean isMergeTransitionComplete =
+        isMergeTransitionComplete(recentChainData.getChainHead());
+    if (!isMergeTransitionComplete
+        && spec.isMilestoneSupported(SpecMilestone.BELLATRIX)
```

Comment:
i think `spec.isMilestoneSupported(SpecMilestone.BELLATRIX)` is redundant since we already check bellatrix spec config at line 75